### ISSUE TITLE
USWDS - Testing: Add ability to test subpages ie Component subpages.

### ISSUE THREAD

**Issue by mejiaj** [uswds/uswds-site#1090]
## Description

Our accessibility script takes an array of URLs. So if we wanted to test all components, we'd have to include all of them. Instead, we should be able to use something like globbing to ensure all subpages are tested.

Currently:
```js
const PAGES = [
  '/',
  '/page-templates/landing/',
  '/page-templates/docs/',
];
```
**Source:** [run-axe.js:13](https://github.com/uswds/uswds-site/commit/c863e0d48b26ce5ad7e9b5c4954df1f1c1733b0f)

Prevents the `run-axe.js` file from becoming unwieldy.

## Additional information [optional]

* Use `npm run axe` to run a11y tests.

**Comment by thisisdano** [comment]
What would this take to complete? I love the idea of using globs, but I also don't really care about having an unwieldy config file at the moment
